src/Pimple/KafkaSchemaRegistryApiClientProvider.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Jobcloud\KafkaSchemaRegistryClient\Pimple;
+
+use Buzz\Client\Curl;
+use Jobcloud\KafkaSchemaRegistryClient\HttpClient;
+use Jobcloud\KafkaSchemaRegistryClient\KafkaSchemaRegistryApiApiClient;
+use LogicException;
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+
+class KafkaSchemaRegistryApiClientProvider implements ServiceProviderInterface
+{
+
+    public const USERNAME = 'kafka_schema_registry_username';
+    public const PASSWORD = 'kafka_schema_registry_password';
+    public const BASE_URL = 'kafka_schema_registry_base_url';
+    public const CLIENT = 'kafka_schema_registry_client';
+    public const REQUEST_FACTORY = 'kafka_schema_registry_request_factory';
+    public const HTTP_CLIENT = 'kafka_schema_registry_http_client';
+    public const API_CLIENT = 'kafka_schema_registry_api_client';
+
```

Comment:
i would really like to keep the structure we have right nowl:
```
'kafka_schema_registry' => [
                'url' => 'broker',
                'username' => 'user',
                'password' => 'pw',
            ],
```

Reply:
This made much problems, because Pimple Container does not support functionality of Service container in sub-array.

Meaning if I define $container[x][y] = function() { << some instantiating code that returns object >> } ,

if I call $container[x][y], it will return me a Closure (function) , not an instantiated object, unless I call function and pass params, which complicates things a lot. Makes lot of code.

Solution (beside magic) would be:
1. Making each service on root level
2. Not giving user possibility to make it own ex. Error Handler, httpclient...